index.html

```diff
@@ -0,0 +1,170 @@
+						Praise & Worship.
+					</p>
+				</div>
+				<div class="event__date">
+					<p class="event__date--tframe">2021-11-19(THU) ~ 20(FRI)</p>
+					<p class="event__date--location">@ Montego Bay Convention Centre</p>
+				</div>
+			</div>
+		</section>
+		<section class="program" id="program">
+			<h2 class="program__header">Main Program</h2>
+			<hr class="program__line" />
+			<ul class="program__list">
+				<li class="program__item">
+					<div class="program__item--icon">
+						<i class="fa fa-church fa-2x program__item--icon"></i>
+					</div>
+					<h3 class="program__item--title">Service</h3>
+					<p class="program__item--desc">
+						Elevate your understanding of Gods word by joining our services.
+					</p>
+				</li>
+				<li class="program__item">
+					<div class="program__item--icon">
+						<i class="fa fa-bible fa-2x program__item--icon"></i>
+					</div>
+					<h3 class="program__item--title">Study</h3>
+					<p class="program__item--desc">
+						Come discuss, and study Gods word with others who seek
+						understanding.
+					</p>
+				</li>
+				<li class="program__item">
+					<div class="program__item--icon">
+						<i class="fa fa-campground fa-2x program__item--icon"></i>
+					</div>
+					<h3 class="program__item--title">Village</h3>
+					<p class="program__item--desc">
+						Bring your kids to our interactive and fun children villages.
+					</p>
+				</li>
+				<li class="program__item">
+					<div class="program__item--icon">
+						<i class="fa fa-pray fa-2x program__item--icon"></i>
+					</div>
+					<h3 class="program__item--title">Prayer</h3>
+					<p class="program__item--desc">
+						Join us in our Prayer Meetings where we converse with our Lord.
+					</p>
+				</li>
+			</ul>
+			<button type="button" class="program__join">Join FALAC Now</button>
+		</section>
+		<section class="speaker" id="speakers">
+			<h2 class="speaker__header">Featured Speakers</h2>
+			<hr class="speaker__line" />
+			<div class="speaker__cards">
+				<template class="speaker__template">
+					<div class="speaker__card">
+						<div class="speaker__img">
+							<img src="images/profilepic/pexels-andrea-piacquadio-3789888.jpg" alt="speaker image"
+								class="speaker__pic" />
+						</div>
+						<div class="speaker__content">
+							<p class="speaker__name">Martin Roon</p>
+							<p class="speaker__qual">Author</p>
+							<hr class="speaker__line2" />
+							<p class="speaker__desc">
+								Martin Roon shares captivating stories of a powerful God with churches,
+								conferences, prisons, and other venues around the country,
+								year-round.
+							</p>
+						</div>
+					</div>
+				</template>
+			</div>
+		</section>
+		<div class="partner desktop">
+			<h2 class="partner__title">Partner</h2>
+			<hr class="partner__line">
+			<div class="partner__cont">
+				<div class="partner__logo"><img src="images/logos/nike-6.svg" alt="nike logo"
```

Comment:
- [ ] These logos should be smaller, you should resize them as given in the design.